bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJetsToQQ_5f_NLO_FXFX/WJetsToQQ_5f_NLO_FXFX_run_card.dat

```diff
@@ -0,0 +1,144 @@
+#***********************************************************************
+#                        MadGraph5_aMC@NLO                             *
+#                                                                      *
+#                      run_card.dat aMC@NLO                            *
+#                                                                      *
+#  This file is used to set the parameters of the run.                 *
+#                                                                      *
+#  Some notation/conventions:                                          *
+#                                                                      *
+#   Lines starting with a hash (#) are info or comments                *
+#                                                                      *
+#   mind the format:   value    = variable     ! comment               *
+#***********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of events (and their normalization) and the required          *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  1500 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+    20 = nevt_job! Max number of events per job in event generation. 
+                 !  (-1= no split).
+average = event_norm ! Normalize events to sum or average to the X sect.
```

Comment:
also here https://github.com/cms-sw/genproductions/pull/1621/files#diff-2c6138a3a9aaa18acaa48176ae3137dbR32

Reply:
These changes were already done in the "bias" cards. So if you see the pull request contains 4 folders: 2 are Zjets and Wjets at NLO but with no bias implementation and the other 2 are with bias i.e. these ones:

https://github.com/cmantill/genproductions/tree/mg26x/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZJetsToQQ_5f_NLO_FXFX_bias

and 

https://github.com/cmantill/genproductions/tree/mg26x/bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJetsToQQ_5f_NLO_FXFX_bias